### PR TITLE
upd(libicu-deb): `71.1` -> `72.1`

### PR DESCRIPTION
**Pull request by Zahrun**

### CODE REVIEW

File: packages/libicu-deb/libicu-deb.pacscript [hunk 1]
```diff
@@ -0,0 +1,9 @@
+name="libicu-deb"
+gives="libicu"
```
Comment (Henryws) [inline packages/libicu-deb/libicu-deb.pacscript:2]:
```suggestion
gives="libicu72"
```

**Review by Henryws — CHANGES_REQUESTED** [review]

File: packages/libicu-deb/libicu-deb.pacscript [hunk 2]
```diff
@@ -0,0 +1,9 @@
+name="libicu-deb"
+version="72.1"
+gives="libicu${version:0:2}"
+url="http://archive.ubuntu.com/ubuntu/pool/main/i/icu/libicu72_${version}-3ubuntu2_amd64.deb"
```
Comment (Henryws) [inline packages/libicu-deb/libicu-deb.pacscript:4]: You should probably add the string splicing to this as well

**Review by Henryws — CHANGES_REQUESTED** [review]

File: packages/libicu-deb/libicu-deb.pacscript (hunk 2 above)
Comment (Zahrun) [reply packages/libicu-deb/libicu-deb.pacscript:4]: Yeah but pacup fails to parse it :/
So anyway I would have to change it manually

Comment (Henryws) [reply packages/libicu-deb/libicu-deb.pacscript:4]: Oh true..

**Pull request merged**